LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -18,6 +18,18 @@ export trixi_is_finished,
 export trixi_step,
        trixi_step_cfptr,
        trixi_step_jl
+export trixi_ndims,
+       trixi_ndims_cfptr,
+       trixi_ndims_jl
+export trixi_nelements,
+       trixi_nelements_cfptr,
+       trixi_nelements_jl
+export trixi_nvariables,
+       trixi_nvariables_cfptr,
+       trixi_nvariables_jl
+export trixi_get_cell_averages,
+       trixi_get_cell_averages_cfptr,
+       trixi_get_cell_averages_jl
```

Comment:
```suggestion
export trixi_load_cell_averages,
       trixi_load_cell_averages_cfptr,
       trixi_load_cell_averages_jl
```
I like the idea from Julia to use `load`/`store` to underline that something "heavy" is transferred. `get`/`set` could also be used for just passing around pointers, while `load`/`store` tell me that this means that a non-negligible amount of memory is likely to be shifted around.

But this is really only a suggestion and depends a lot on personal preference, so proceed as you see fit